jobs/management/commands/populate_database.py

```diff
@@ -0,0 +1,152 @@
+                description = ''.join(selected_lines)
+    else:
+        description = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.'
+    return description
+
+
+class Command(NoArgsCommand):
+    help = "Populates database with meetups, companies and job offers."
+
+    def add_meetup(self, title, city, country, description, meetup_date):
+        meetup = Meetup.objects.get_or_create(
+            title=title,
+            contact_email='example@example.com',
+            city=city,
+            country=country,
+            description=description,
+            ready_to_publish=True,
+            meetup_date=meetup_date,
+        )
+        return meetup
+
+    def add_company(self, name, website):
+        company = Company.objects.get_or_create(
+            name=name,
+            website=website
+        )
+        return company
+
+    def add_job(self, title, company, city, country, description):
+        job = Job.objects.get_or_create(
+            title=title,
+            company=company,
+            contact_email='example@example.com',
+            city=city,
+            country=country,
+            description=description,
+            ready_to_publish=True,
+        )
+        return job
+
+    def handle_noargs(self, **options):
+        # Adding some meetups with future dates.
+        self.add_meetup(
+            title='Django Girls Warsaw',
+            city='Warsaw',
+            country='PL',
+            description=read_lines(),
+            meetup_date='2015-04-01'
+        )
+        self.add_meetup(
+            title='Women in Technology',
+            city='London',
+            country='GB',
+            description=read_lines(),
+            meetup_date='2015-05-15'
+        )
+        self.add_meetup(
+            title='Learn javascript',
+            city='Paris',
+            country='FR',
+            description=read_lines(),
+            meetup_date='2015-06-12'
+        )
+
+        self.add_meetup(
+            title='Python breakfast',
+            city='Berlin',
+            country='DE',
+            description=read_lines(),
+            meetup_date='2015-07-01'
+        )
+
+        self.add_meetup(
+            title='Girls Meetup',
+            city='New York',
+            country='US',
+            description=read_lines(),
+            meetup_date='2015-08-01'
+        )
+
+        # Adding some companies.
+        self.add_company(
+            name='Google',
+            website='http://www.google.pl/about/careers/students/'
+        )
+
+        self.add_company(
+            name='Amazon',
+            website='http://www.amazon.jobs/team-category/university-recruiting'
+        )
+
+        self.add_company(
+            name='Digital Ocean',
+            website='https://careers.digitalocean.com/'
+        )
+
+        # Adding some job offers.
+        self.add_job(
+            title='Intern',
+            company=Company.objects.get(name='Google'),
```

Comment:
If you assign the companies to a variable, you can use them again when you do add_job, instead of re-querying the database every time (i.e. comany=google instead of Comany.objects.get....)